python/pyiceberg/transforms.py

```diff
@@ -709,8 +709,11 @@ def _(_type: IcebergType, value: int) -> str:
 class UnknownTransform(Transform[S, T]):
     """A transform that represents when an unknown transform is provided
     Args:
-      source_type (IcebergType): An Iceberg `Type`
       transform (str): A string name of a transform
+
+    Keyword Args:
+      source_type (IcebergType): An Iceberg `Type`
```

Comment:
I think it is better to just drop this one. An `UnknownTransform` doesn't really has a `source_type` since it is just a fallback transform in case we get an unexpected transform.